src/main/java/com/untamedears/jukealert/model/appender/BroadcastEntryAppender.java

```diff
@@ -50,18 +43,16 @@ public void acceptAction(SnitchAction action) {
 				continue;
 			}
 			if (snitch.hasPermission(uuid, JukeAlert.getInstance().getPermissionHandler().getSnitchAlerts())) {
-				TextComponent comp = log.getChatRepresentation(player.getLocation(), true);
-
-				if (settings.shouldShowDirections(uuid)) {
-					comp.addExtra(String.format("  %s", JAUtility.genDirections(snitch, player)));
-				}
+				TextComponent comp = log.getChatRepresentation(snitch.getLocation(), true);
```

Comment:
One issue I'm not entirely sure of the implications is this in classics/master branch uses the player location and here I've used the snitch location in order to make it work/compile. This doesn't present any immediate issues in game however there is probably a reason to use the players location

Reply:
Looks like one implication can be fond in the log.getChatRepresentation function; there is conditional logic to print out the location passed if it isn't the snitch location. In this case, you're always passing the snitch location, so that logic will never occur. 

I'd add a //TODO here to indicate it should, ideally, be a world location. You could probably "ask" Zeus where the location is, the getChatRepresentation already has logic to decide if the passed location is in the same world or not, etc.... otherwise seems OK.